commands/utils/archive.js

```diff
@@ -71,6 +71,24 @@ function archive_project(lt_config) {
       { cwd: process.cwd(), ignore: ignore_files },
       { prefix: "project/" }
     );
+    //OverRide NPM Dependencies
+    if (lt_config["run_settings"]["npm_dependencies"]) {
+      console.log("Overriding NPM Dependencies");
+      let rawdata = fs.readFileSync("package.json");
+
+      let package = JSON.parse(rawdata);
```

Comment:
Add a try-catch block while using JSON.parse

Reply:
The json validation is already happening in a different file and code will not reach here is parsing fails, that's why error handling is not required here

Reply:
It’s still a safe check to add :)